transactions/idTableStaking/scripts/get_approved_but_not_staked_nodes.cdc

```diff
@@ -0,0 +1,20 @@
+import FlowIDTableStaking from 0xIDENTITYTABLEADDRESS
+
+// This script returns the list of nodes that are on the approved list but are not currently staked.
+pub fun main(): [String] {
+    let approvedIDs = FlowIDTableStaking.getApprovedList()
+    let stakedIDs = FlowIDTableStaking.getStakedNodeIDs()
+
+    let stakedIDsMap: {String: Bool} = {}
+    for stakedID in stakedIDs {
+		stakedIDsMap[stakedID] = true
+	}
```

Comment:
Could you take a pass over the indentation in the script before merging?

Reply:
yeah - fixing the indentation right now